pkg/rest/rest.go

```diff
@@ -0,0 +1,21 @@
+package rest
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+// ReadBody reads body from a ReadCloser and returns it as a string
+func ReadBody(body io.ReadCloser) string {
+	buf := new(bytes.Buffer)
+	buf.ReadFrom(body)
+	return buf.String()
+}
+
+// CloseResponse reads the body and close the response. To be used to prevent file descriptor leaks.
+func CloseResponse(response *http.Response) {
+	ioutil.ReadAll(response.Body)
+	response.Body.Close()
+}
```

Comment:
you don't handle the errors at all here

Reply:
oh, I just copied it from https://github.com/codeready-toolchain/devcluster/blob/6defc6cce09762e3b65af412d20ba4970dd30456/pkg/rest/rest.go 🙂  and figured errors are ignored because it's mainly for cleanup

Reply:
yeah. we don't really care about errors there. We could log them though. Or at least make the linters happy by using _

Reply:
Updated the `ReadBody` func to return the error and let the caller decide what to do.
Updated the `CloseResponse` func by using `_` to make the linters happy